media-libs/libopenshot/libopenshot-0.1.7.ebuild

```diff
@@ -71,9 +74,8 @@ src_configure() {
 }
 
 src_test() {
-	pushd "${BUILD_DIR}/tests" > /dev/null || die
-	./openshot-test || die "Tests failed"
-	popd > /dev/null || die
+	cd "${BUILD_DIR}" || die
+	emake test
```

Comment:
Just so I'm sure: you mean change directory and then call `cmake-utils_src_test`, right?